keel-api/keel-api.gradle.kts

```diff
@@ -0,0 +1,5 @@
+plugins {
+  `java-library`
+}
+
+// DO NOT ADD ANY DEPENDENCIES HERE!
```

Comment:
Could you also add a brief explanation of why so that this comment is more likely to be respected? :)

Reply:
Done